dp_wizard/utils/code_generators/abstract_generator.py

```diff
@@ -221,7 +221,9 @@ def _make_partial_stats_context(self):
             if has_bins(get_analysis_by_name(plan[0].analysis_name))
         ]
 
-        privacy_unit_block = make_privacy_unit_block(self.analysis_plan.contributions)
+        privacy_unit_block = make_privacy_unit_block(
+            contributions=self.analysis_plan.contributions, contributions_entity="TODO"
```

Comment:
How would you like to handle this TODO here? or it was ignored

Reply:
Sorry to miss this one.
```suggestion
            contributions=self.analysis_plan.contributions,
            contributions_entity=self.analysis_plan.contributions_entity,
```